mantidproject/mantid
# Add vulcan IDFs

**Description of work.**

Added 2 IDF for VULCAN since 2017.



**To test:**

Compare LoadEventNexus with flag "LoadNexusInstrumentXML" on and off for any run before March 2019.  For example, 
    /SNS/VULCAN/IPTS-22752/nexus/VULCAN_172440.nxs.h5

Also check whether all the associated tests are passed.

*There is no associated issue.*

Change is added to release note (diffraction)

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
